### ISSUE TITLE
Fix: the Navbar Collision in responsive View(Logo and About-us tab) 

### ISSUE THREAD

**Issue by prabhat04011998** [anitab-org/anitab-org.github.io#166]
### Describe the bug
The Responsive view of website leads to distortion in navbar(Header).

### To Reproduce
1. Go to https://anitab-org.github.io/
2. Click on inspect->mobile view icon
3. Scroll up (Upto top)
4. See error( logo and about us us tab colliding)

### Expected behavior
This about us and logo should not collide.

### Screenshots

![issue_in_responsive_view](https://user-images.githubusercontent.com/28345582/100468677-e660e880-30fa-11eb-8ab4-3d6c2c1884a0.png)

### Desktop (please complete the following information):
 - OS: [Linux 20+]
 - Browser [Firefox]
 - Version [83.0]

### Smartphone (please complete the following information):
 - Device: [Xiomi ]
 - OS: [android]
 - Browser [chrome]
 - Version [latest]

### Additional context
It's the issue of front-end (User Interface).
Some styling needs to fix.

**Comment by nandini45** [comment]
@prabhat04011998  do u want to work on it or any newcomer can work on it?

**Comment by prabhat04011998** [comment]
@nandini45 yes i can work over it ,please assign it to me!

**Comment by nandini45** [comment]
@prabhat04011998   if the time limit to do the work exceed 3 days
do give us update. happy coding

**Comment by prabhat04011998** [comment]
@nandini45  okay i'll update.

**Comment by tichnas** [comment]
@nandini45 @prabhat04011998 just a suggestion from me, I think this issue is more about a responsive navbar (which gets hidden on the smaller screen and shows when touched on a hamburger icon). Just handling the about us and logo may still cause issues on smaller screens or when there are more pages.

**Comment by nandini45** [comment]
@tichnas ok so as far as for more other pages its considered i can't say anything. But here to be true  idk the exact solution
Do u think aligning will help or we have to redesign some bit,??

**Comment by tichnas** [comment]
@nandini45 I don't think changing the measurements will help in this case (even with the current no. of pages).

I tried after removing all the empty space in the navbar (which is extreme, and obviously not look good) and the minimum width it still requires is 440 px with the look as:
![image](https://user-images.githubusercontent.com/56799401/100781222-c5690200-3430-11eb-8bd9-542f7ef81555.png)

Given that smaller screen devices go up to a minimum of 320 px (iPhone 5), it's not possible to keep a good looking navbar just by changing the measurements.

One possible solution which most of the websites take is simply don't show the full navbar on smaller screens. I can't explain much in words what I mean, but please take a look at the navbar of https://tichnas.github.io/ (on smaller screens)

**Comment by prabhat04011998** [comment]
Hey @nandini45 
here are the some Snaps of different pages please review once.
![image](https://user-images.githubusercontent.com/28345582/100868165-3eb03580-34c1-11eb-863f-21309e4671f2.png)

**Comment by nandini45** [comment]
@tichnas  yes i get ur point. but then this issue is a total different one. 
 i like what you did if that can happen well and good. and one more point if you want to do it create other issue or ask the person working on this issue to do so if he can.

**Comment by nandini45** [comment]
@prabhat04011998  great job. if you can make the changes that @tichnas  proposed it will be a lot of help.
.

**Comment by prabhat04011998** [comment]
@nandini45  thanks for the review,

@tichnas  please confirm i understand your point right.

so our navbar is fixed right now, like it shows partially on the small screen and need to scroll right , 
what is you suggestion that we develop this navbar or new navbar collapsible,
like you showed in the example.
like on small scree a single menu item will show , on click over it the list of tabs will appear right?

basically COLLAPSIBLE NAVBAR you are suggesting.

**Comment by tichnas** [comment]
@prabhat04011998 Yes, you understood correctly :+1:

**Comment by prabhat04011998** [comment]
@tichnas okay i understand , but it's a  whole new issue, 
I think you can create a new issue and please assign it to me!

**Comment by tichnas** [comment]
@prabhat04011998 Ok, let me create one...

**Comment by Bucky25** [comment]
@tichnas @tichnas we should think of collapsible nav bar as the scroll on mobile or small screen does not look good only nav bar remains of full width and rest things are of width of small screen 
we can make a new issue and have @prabhat04011998 assign it

**Comment by Bucky25** [comment]
![WhatsApp Image 2020-12-08 at 2 25 34 AM(1)](https://user-images.githubusercontent.com/65112369/101404584-d74d1800-38fc-11eb-8a5d-62336b9cf103.jpeg)
![WhatsApp Image 2020-12-08 at 2 25 34 AM(2)](https://user-images.githubusercontent.com/65112369/101404590-d916db80-38fc-11eb-8468-b7232588bc26.jpeg)
![WhatsApp Image 2020-12-08 at 2 25 34 AM](https://user-images.githubusercontent.com/65112369/101404596-dae09f00-38fc-11eb-98c5-239dc232646c.jpeg)
here are screenshots what i am trying to say

**Comment by tichnas** [comment]
@Bucky25 @prabhat04011998 Sorry I got busy and forgot to create the issue, but I've created one now.

**Comment by prabhat04011998** [comment]
@nandini45  PR for this issue is already merged we can close this.

**Comment by nandini45** [comment]
@prabhat04011998 okk thanks for the work
